pkg/operator/ceph/nfs/spec.go

```diff
@@ -200,6 +201,7 @@ func (r *ReconcileCephNFS) daemonContainer(nfs *cephv1.CephNFS, cfg daemonConfig
 			"-F",           // foreground
 			"-L", "STDERR", // log to stderr
 			"-p", ganeshaPid, // PID file location
+			"-N", logLevel, // Change Log level
```

Comment:
The logLevel in the CR might not be set, so if it's empty let's set a default or else the pod will likely fail.

Reply:
Done.